components/editor/overview.md

```diff
@@ -86,15 +69,20 @@ The Telerik UI for Blazor Editor uses the ProseMirror engine and it depends on i
 
 The Blazor HTML Editor interacts with its content (value) like all standard components - through its `Value` parameter. You can use it to get and set the HTML string the editor will work with. You can read more about value binding and data binding [here]({%slug get-started-value-vs-data-binding%}).
 
-You can use the following features to get or set the editor content:
+#### You can use the following features to get or set the editor content:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
 
-* `@bind-Value` - the recommended approach of using two-way binding to get and set the content of the editor. It lets your view-model provide the initial value, and it will update the view-model as the user alters the HTML.
+| Parameter | Type and Default value | Description |
+|-----------|------------------------|-------------|
+| `@bind-Value`  | `EventCallback<string>` | The recommended approach of using two-way binding to get and set the content of the editor. It lets your view-model provide the initial value, and it will update the view-model as the user alters the HTML. |
```

Comment:
`@bind-Value` is not a parameter. `Value` is a parameter and `@bind` is a directive. We can just mention which parameters support two-way binding and link to the respective `...Changed` event.